src/Validation/PlValidation.php

```diff
@@ -89,22 +85,19 @@ public static function nip($check)
      * @return bool Success.
      * @link http://pl.wikipedia.org/wiki/PESEL
      */
-    public static function pesel($check)
+    public static function pesel(string $check): bool
     {
-        $pattern = '/^[0-9]{11}$/';
+        $pattern = '/^\d{11}$/';
         if (preg_match($pattern, $check)) {
             $sum = 0;
             $weights = [1, 3, 7, 9, 1, 3, 7, 9, 1, 3];
-
             for ($i = 0; $i < 10; $i++) {
                 $sum += $check[$i] * $weights[$i];
             }
-
             $control = 10 - $sum % 10;
-            if ($control == 10) {
+            if ($control === 10) {
                 $control = 0;
             }
-
             if ($check[10] == $control) {
```

Comment:
=== ??
$check is not string ?

Reply:
jep, could probably be a stringish vs int value comparison.